### PR TITLE
CB-8404 Rotate camera feed with device orientation

### PR DESCRIPTION
**Pull request by muratsu**
Fixed 2 bugs in windows where we have trouble rotating the image on preview and saving

### CODE REVIEW

File: src/windows/CameraProxy.js [hunk 1]
```diff
@@ -310,29 +319,50 @@ function takePictureFromCameraWP(successCallback, errorCallback, args) {
 
         tempFolder.createFileAsync(fileName, generateUniqueCollisionOption)
             .then(function(tempCapturedFile) {
-                capturedFile = tempCapturedFile;
-                return capture.capturePhotoToStorageFileAsync(encodingProperties, capturedFile);
+                return new WinJS.Promise(function (complete) {
+                    var imgStream = new Windows.Storage.Streams.InMemoryRandomAccessStream();
```
Comment (nikhilkh) [inline src/windows/CameraProxy.js]: imgStream.close() needs to be invoked once you are done using it to release system resources. It needs to be part of the success and error case.

Comment (nikhilkh) [inline src/windows/CameraProxy.js]: Same needs to be done for `fileStream`.

Comment (nikhilkh) [inline src/windows/CameraProxy.js]: Some comments on why the rotation needs to be done will be good - since its not quite obvious.

**Comment by nikhilkh** [comment]
It might be a good idea to file a JIRA to improve the code for resizeImage and not to use canvas for a resize operation.

**Comment by muratsu** [comment]
Rebased and squashed - ready to merge.

**Pull request merged**